content/en/storage-providers/operate/addresses.md

```diff
@@ -33,7 +33,7 @@ The owner address corresponds to a Lotus node address provided during the miner
 - Withdrawing balance from the _miner actor_.
 - Submit _WindowPoSts_, **unless _control addresses_ are defined and have enough balance** (continued below).
 
-The address chosen to be the miner's _owner address_ is designed to be kept offline in _cold storage_, or backed up by a [hardware wallet]({{< relref "manage-fil" >}}). In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
+The address chosen to be the miner's _owner address_ is designed to be kept offline in _cold storage_, or backed up by a [hardware wallet](https://lotus.filecoin.io/lotus/manage/manage-fil/#ledger). In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
```

Comment:
Let's not use a direct link but a relref in case content is moved, it can still be found using aliases.

Reply:
Removed the links and added a section under Managing balances about Fees and good practices. 
I made short a example for now, after we have moved Fees we can direct to that instead.